app/library/postgres.py

```diff
@@ -254,6 +280,7 @@ async def delete_scene_from_postgres(scene_id: str, session):
                 await update_scenario_from_postgres(
                     scenario.id, scenario.as_dict(), session
                 )
+                break
```

Comment:
is this because a scenario will only refer to a scene once?

Reply:
Yup exactly, no need to go through the rest of the scenario's scene ids!